lib/Migrator/overlay4.json

```diff
@@ -1342,5 +1342,5 @@
     "RightUsr": "",
     "RightComment": "firstIndex",
     "ModuleName": "Swift"
-  },
+  }
```

Comment:
Why are there changes in the `json` files?

Reply:
My editor did that because trailing comma. I will remove that. Sorry!